app/controllers/users.js

```diff
@@ -14,3 +16,25 @@ exports.userRegistration = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.userLogIn = (req, res, next) => {
+  logger.info('POST method start. User authentication.');
+  const user = req.body;
+  let storedUser = {};
+  return users
+    .getUserByEmail(user.email)
+    .then(response => {
+      if (response) {
+        storedUser = response.dataValues;
+        return helper.comparePassword(user.password, storedUser.password);
+      }
+      return Promise.reject(errors.badLogInError(`User with email ${user.email} not found`));
```

Comment:
For security, is better to not specify email error or password error. Just return invalid credentials or something like that.